controllers/git_tekton_resources_renovater.go

```diff
@@ -233,7 +233,7 @@ func generateConfigJS(slug string, repositories []renovateRepository) string {
 				branchPrefix: "rhtap/",
 				commitMessageExtra: "",
 				commitMessageTopic: "RHTAP references",
-				prFooter: "",
+				prFooter: "To execute Skipped RHTAP Pipeline write comment ` + "`/ok-to-test`" + `",
```

Comment:
RHTAP shouldn't be used by itself. Instead, we should spell out the whole phrase. If we want to get away from the specific naming of the service, we can also try to generalize it. Maybe something like calling them test pipelines?

RHTAP == Red Hat Trusted Application Pipeline, so this would also read `To execute skipped Red Hat Trusted Application Pipeline pipeline [...]` which I think is unneeded redundancy.

Reply:
Updated.